SubmittedAssignments/Sanket_Jadhav/preview.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE+ html>
+<html>
+
+<head>
+	<link rel="stylesheet" type="text/css" href="./css/style.css">
+	<link rel="stylesheet" type="text/css" href="./css/bootstrap.min.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Alex+Brush&display=swap" rel="stylesheet">
+</head>
+
+<body>
+	<!--Certificate Wrapper-->
+	<div class="certificate-wrapper">
+				
+
+		<!--Certificate background-->
+		<div class="certificate-background">
+					
+				    	
+		</div>
+
+		<!--Content of Certificate-->
+		<div class="certificate-content">
+			<div class="outerborder">
+				<div class="innerborder">
+					<div class="title">C E R T I F I C A T E</div>
+					<div class="recttop"></div>
+					<div class="rectleft"></div>
+					<div class="recttop1"></div>
+					<div class="recttop2"></div>
+					<div class="recttop3"></div>
```

Comment:
the design class should be out side in content class (add in div class="certificate-background")